### DISCUSSION TITLE
Odd issue with agent status displaying incorrectly

### DISCUSSION THREAD

**Discussion by LVInternetworks** [amidaware/tacticalrmm#1546] [Q&A]
Recently renew'd ssl certificate ( might or might not be relevant ) and rebooted host. Upon returning all agents now show as "overdue" however I can still remote to them. 

If i try refreshing the agent staus I get " Unable to contact agent" 400: Bad Request.

Remote background works somewhat. I can see drives and access terminal, but I cannot see retrieve any services or process's or view the event log.

I am just confused beyond belief right now. Has anyone had an issue like this? 

Also I tried reinstalling one of the agents and originally it was showing as online but Boot Time was listed as 54 years ago. It was not grabbing the logged in user so after  a reboot of the desktop its now showing as overdue again and the agent version is showing as 0.1.0 ?

**Comment by wh1te909** [comment]
this happens when you either didn't renew cert properly, or renewed but didn't update permissions of the cert file (did you follow the docs and run the update script with the `--force` flag after renewing?

`sudo systemctl status nats` will probably show you the error

**Reply by LVInternetworks** [reply]
Ok thank you will try that and report back!
